nncf/torch/quantization/prepare_for_inference.py

```diff
@@ -0,0 +1,146 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+
+import numpy as np
+import torch
+from torch.quantization.fake_quantize import FakeQuantize
+
+from nncf.torch.nncf_network import NNCFNetwork
+from nncf.torch.quantization.layers import AsymmetricQuantizer
+from nncf.torch.quantization.layers import BaseQuantizer
+from nncf.torch.quantization.layers import SymmetricQuantizer
+
+
+def replace_quantizer_to_torch_native_module(model: NNCFNetwork) -> None:
```

Comment:
Make this return the `model`. I know that the function works in-place, but it is more evident that this function modifies the object by making it return the `NNCFNetwork` as if it were a new object.